app/controllers/api/teams.php

```diff
@@ -762,14 +763,20 @@
             throw new Exception('Team not found', 404);
         }
 
-        if (!$projectDB->deleteDocument($membership->getId())) {
+        try {
+            $projectDB->deleteDocument($membership->getId());
+        } catch (AuthorizationException $exception) {
+            throw new Exception('Unauthorized permissions', 401);
+        } catch (\Exception $exception) {
             throw new Exception('Failed to remove membership from DB', 500);
         }
 
         if ($membership->getAttribute('confirm')) { // Count only confirmed members
+            Authorization::disable();
```

Comment:
This looks solid. As @Meldiron suspected, we have moved away from `disable` and `reset` methods to use the new `skip` method which is considered safer.

Reply:
This `skip()` function is a great addition, but I couldn't use it because it seems like that that function is in the db-alpha branch rather than master.